haec-user-service/src/modules/user/controllers/user.controller.ts

```diff
@@ -19,6 +19,16 @@ type VerifyUserDto = {
   password: string;
 };
 
+type LoginUserDto = {
+  email: string;
+  password: string;
+};
+
+type LogoutUserDto = {
+  email: string;
+  password: string;
```

Comment:
A user that's logging out shouldn't be passing a password

Reply:
But how will we know that this user has credentials to log himself out?

Just kidding. Makes sense.